presto-hive/src/main/java/com/facebook/presto/hive/StoragePartitionLoader.java

```diff
@@ -259,7 +261,7 @@ public StoragePartitionLoader(
 
             return addSplitsToSource(splits, splitFactory, hiveSplitSource, stopped);
         }
-        PathFilter pathFilter = isHudiParquetInputFormat(inputFormat) ? hoodiePathFilterLoadingCache.getUnchecked(configuration) : path1 -> true;
+        PathFilter pathFilter = path1 -> true;
```

Comment:
Yes. I kept it to be compatible in case users disable and not use `HudiDirectoryLister`. But, actually we introduced the path filter in #13818 . Let's remove it in a follow up PR.